### PR TITLE
Truncating long protein change values

### PR DESCRIPTION
**Pull request by onursumer**
# What? Why?
Fix https://github.com/cBioPortal/cbioportal/issues/2871

# Checks
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

### CODE REVIEW

**Comment by onursumer** [comment]
@adamabeshouse I was thinking the `maxLength` as the maximum number of characters excluding the suffix. Actually there is also a `buffer` property for a similar purpose, but they are not the same thing.

**Comment by adamabeshouse** [comment]
I think in general it would make more sense to do that inside the component, because then you can make most use of the space, but it doesn't matter that much.

File: src/shared/components/TruncatedText.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
-import {If, Else, Then} from 'react-if';
+import classNames from 'classnames';
```
Comment (alisman) [inline src/shared/components/TruncatedText.tsx]: not a big deal but in future lets keep it lowercase in keeping with module name

File: src/shared/components/TruncatedText.tsx [hunk 2]
```diff
@@ -45,7 +46,10 @@ export default class TruncatedText extends React.Component<ITruncatedTextProps,
     public render()
     {
         let content = (
-            <span style={{whiteSpace: "nowrap"}}>
+            <span
+                style={{whiteSpace: "nowrap"}}
+                className={classNames(this.props.className)}
```
Comment (alisman) [inline src/shared/components/TruncatedText.tsx]: is classNames even necessary here?

File: src/shared/components/TruncatedText.tsx (hunk 1 above)
Comment (onursumer) [inline src/shared/components/TruncatedText.tsx]: oops, this was a `copy&paste` from another file, apparently we have it like this at many places already.

File: src/shared/components/TruncatedText.tsx (hunk 2 above)
Comment (onursumer) [inline src/shared/components/TruncatedText.tsx]: hmm, probably not. I used it to handle the case when className is `undefined`.

**Comment by onursumer** [comment]
@adamabeshouse it might be better to improve `TruncatedText` in another PR, since it has been already used by other formatters.

Not related to this PR, but I also think that `<TruncatedText>{text}</TruncatedText>` looks better than `<TruncatedText text={text} />`.

**Pull request merged**